daisy_workflows/build-publish/windows_media/components/common/logging.psm1

```diff
@@ -0,0 +1,88 @@
+<#
```

Comment:
Is there a reason we're not using https://github.com/GoogleCloudPlatform/compute-image-windows/blob/master/powershell/gce_base.psm1?

Reply:
It's just existing code. Copied to this place to ensure they have the same functionality. Optimizing it is a separate thing in my opinion.